include/interface/blas1_interface.h

```diff
@@ -76,6 +76,33 @@ template <typename executor_t, typename container_0_t, typename container_1_t,
 typename executor_t::policy_t::event_t _dot(
     executor_t &ex, index_t _N, container_0_t _vx, increment_t _incx,
     container_1_t _vy, increment_t _incy, container_2_t _rs);
+
+/**
+ * \brief Computes the inner product of two vectors with extended precision
```

Comment:
Does extended precision just mean double precision?

Reply:
Yes, I think so. To be honest I just got the function description from _dot() since it's essentially the same function. Happy to update both descriptions if you think it's more clear

Reply:
Done